printer_mods/README.md

```diff
@@ -160,6 +160,6 @@ like so:
 | | [V0 PiCam Mount](./xbst_/V0_Picam)| A Raspberry Pi Camera Mount For V0 | :heavy_check_mark: | :x: | :x: | :x: |
 | | [DuetZero](./xbst_/DuetZero) | Pi Zero mount over a Duet2 for space saving | :heavy_check_mark: |:heavy_check_mark: |:heavy_check_mark: |:x:|
 | Zen3D | [V0 Handle](./Zen3D/V0_Handle)| Voron0 Handle for 3mm panels | :heavy_check_mark: |:x: |:x: |:x:|
-| zruncho | [V0 No-Drop Nuts](./zruncho/V0_No_Drop_Nuts) | Voron0 nut inserts which auto-align | :heavy_check_mark: | :x: | :x: | :x: |
+| zruncho | [V0 No-Drop Nuts](./zruncho/V0_No_Drop_Nuts) | Voron0 nut inserts for which auto-align (LDO only) | :heavy_check_mark: | :x: | :x: | :x: |
```

Comment:
The `for` doesn't make sense in the description there.  And change `LDO only` to `LDO frame only` or `LDO extrusions only`.